src/components/admin/settings/general/Update.vue

```diff
@@ -54,33 +65,56 @@ const isSubmitting = ref(false);
 const onSubmit = handleSubmit(async values => {
   try {
     isSubmitting.value = true;
+    
+    // Validate and prepare featured links
+    const validLinks = featuredLinks.value
+      .filter(link => link.url && isValidUrl(link.url))
+      .map(link => ({ name: link.name || link.url, url: link.url }));
```

Comment:
it must be validated through zod schema which needs to go into ~/src/schema/settings

Reply:
Yes, the formSchema with toTypedSchema(generalSettingsSchema) is validated using a Zod schema, which is defined in  ~/src/schemas/setting file.

Reply:
do not validate like this. it should go through existing validation pattern only. see: https://vee-validate.logaretm.com/v4/guide/composition-api/nested-objects-and-arrays/

Reply:
updated bro!